src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -0,0 +1,186 @@
+﻿open System
+open System.IO
+open FSharp.Compiler.SourceCodeServices
+open FSharp.Compiler.Text
+open ProjectSystem
+open Argu
+open FSharp.Analyzers.SDK
+open GlobExpressions
+
+type Arguments =
+    | Project of string
+    | Analyzer of string
```

Comment:
Suggestion: rename to `Analyzers_Path` since this is the path of the directory in which a bunch analyzers can be loaded?

Reply:
Done